set.mm

```diff
@@ -22426,6 +22430,63 @@ primitive connectives (no defined terms) on the right-hand side.
       FBTUDCAUCJKLUDBCMNOABCPRS $.
   $}
 
+  $( Absorption of existence condition by uniqueness.  (Contributed by NM,
+     4-Nov-2002.)  Shorten proof and avoid ~ df-eu .  (Revised by BJ,
+     14-Oct-2022.) $)
+  moabs $p |- ( E* x ph <-> ( E. x ph -> E* x ph ) ) $=
+    ( wmo wex wi ax-1 wn nexmo pm2.6 ax-mp impbii ) ABCZABDZLEZLMFMGLENLEABHMLI
+    JK $.
```

Comment:
Suggest to prove this using nexmo, id, ja instead of nexmo, pm2.6, ax-mp

Reply:
Thanks ! `MM-PA> minimize_with *` did not catch that. Do you want I add the "proof shortened by WL" tag ?

Reply:
no